generateDataset.sh

```diff
@@ -9,4 +9,4 @@ fi
 
 "$DIR/gradlew" --stop
 "$DIR/gradlew" clean
-"$DIR/gradlew" -p "$DIR" runRefactoringsExperiments -Prunner=RefactoringsExperiments -PprojectsDirPath="$PWD/$1" -PdatasetsDirPath="$PWD/$2" -PgeneratePositiveSamples -PgenerateNegativeSamples
\ No newline at end of file
+"$DIR/gradlew" -p "$DIR" runRefactoringsExperiments -Prunner=RefactoringsExperiments -PprojectsDirPath="$1" -PdatasetsDirPath="$2" -PgeneratePositiveSamples
```

Comment:
it would be nice to add the information about running this script to README.
it would be useful if a user wants to run it on remote machine, for example

BTW, should we add both options?-PgeneratePositiveSamples -PgenerateNegativeSamples

Reply:
will do!